app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -458,7 +458,7 @@
   </div>
   <div class="field">
     <label>{{t 'Event Topic'}}</label>
-    {{#ui-dropdown class='search selection' selected=data.event.topic onChange=(action (mut data.event.topic)) as |execute mapper|}}
+    {{#ui-dropdown class='search selection' allowAdditions=true selected=data.event.topic.name onChange=(action (mut data.event.topic)) as |execute mapper|}}
```

Comment:
We do not allow additionsFor this. The user must not be able to add their own topic.